[Docs] coupled solid documentation (#1539)
GEOSX/GEOSX: src/coreComponents/constitutive/docs/BiotPorosity.rst

.. _BiotPorosity:

####################################################
Biot Porosity Model
####################################################

Overview
======================

> Accodirng to the poroelasticity theory, the porostiy (pore volume) change with respect to pressure is expressed as the following Biot Porosity Model:

Reviewer: ```suggestion
According to poroelasticity theory, the porosity (pore volume) change with respect to pressure is expressed via the following Biot Porosity Model:
```

.. math::

>    \frac{d{\phi}}{dp} = {\alpha}{\frac{d{\epsilon}}{dp} + \frac{\alpha - \phi}{K_s}

Reviewer: Do we have a reference for this expression?

Reviewer 2: no, but let me find one and add it.

Reviewer: Also, does this expression reflect how the model is implemented?

where :math:`\alpha` is the Biot coefficient, :math:`\epsilon` is the volumetric strain, :math:`K_s` is the bulk modulus of the grains and :math:`p` is the pressure.

Parameters
======================

The Biot Porosity Model can be called in the
``<Constitutive>`` block of the input XML file.

> This permeability model must be assigned a unique name via

Reviewer: ```suggestion
This porosity model must be assigned a unique name via
```
